### PR TITLE
HOFD-9 add save form session & fix acceptance tests

### PR DESCRIPTION
**Pull request by RobertMcCann**
What
Added behaviour that saves form on each page to feature/database
Fixed  broken acceptance tests that were referring to unused save form/ continue save form pages

Why
So user can save their form

### CODE REVIEW

File: test/_features/example_app/database.feature [hunk 1]
```diff
@@ -0,0 +1,48 @@
+@feature @save_and_return
```
Comment (al01010) [inline test/_features/example_app/database.feature:1]: My suggestion might be to name this file save_and_return.feature

File: apps/feature/behaviours/save-form-session.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+module.exports = superclass => class extends superclass {
+  saveValues(req, res, next) {
+    super.saveValues(req, res, err => {
+      if (err) {
+        next(err);
+      }
+      const session = req.sessionModel.toJSON();
+      delete session['csrf-secret'];
+      delete session.errors;
+      axios.post('http://localhost:3000/forms', {
```
Comment (al01010) [inline apps/feature/behaviours/save-form-session.js:15]: could create a baseUrl for the url here at the top, to keep the function clearer.

File: test/_features/example_app/database.feature (hunk 1 above)
Comment (RobertMcCann) [reply test/_features/example_app/database.feature]: changed the file names, thanks

File: apps/feature/behaviours/save-form-session.js (hunk 2 above)
Comment (RobertMcCann) [reply apps/feature/behaviours/save-form-session.js]: created a baseUrl const for the function

**Comment by al01010** [comment]
Looks good to me

**Pull request merged**